README.md

```diff
@@ -99,29 +97,29 @@ To run the project, execute the following command:
 
 👤 **Nilton Segura**
 
-- GitHub: [@Nkaleth](https://github.com/Nkaleth)
-- Twitter: [@NoeSeguraL](https://twitter.com/NoeSeguraL)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/niltonsegura/)
+-   GitHub: [@Nkaleth](https://github.com/Nkaleth)
+-   Twitter: [@NoeSeguraL](https://twitter.com/NoeSeguraL)
+-   LinkedIn: [LinkedIn](https://www.linkedin.com/in/niltonsegura/)
 
 👤 **Hans-Derly Rameau**
 
-- GitHub: [@hansderly](https://github.com/hansderly)
-- Twitter: [@\_hansderly](https://twitter.com/_hansderly)
-- LinkedIn: [ansderly-rameau](https://linkedin.com/in/ansderly-rameau/)
+-   GitHub: [@hansderly](https://github.com/hansderly)
+-   Twitter: [@\_hansderly](https://twitter.com/_hansderly)
+-   LinkedIn: [ansderly-rameau](https://linkedin.com/in/ansderly-rameau/)
 
 👤 **Dennis Every**
 
-- GitHub: [@dennis-every](https://github.com/dennis-every)
-- Twitter: [@dennis_every](https://twitter.com/dennis_every)
-- LinkedIn: [dennis-every](https://www.linkedin.com/in/dennis-every/)
+-   GitHub: [@dennis-every](https://github.com/dennis-every)
+-   Twitter: [@dennis_every](https://twitter.com/dennis_every)
+-   LinkedIn: [dennis-every](https://www.linkedin.com/in/dennis-every/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- none
+-   none
```

Comment:
- [optional] I think your repo will be more informative and engaging if you can think of one or more future feature(s) to add.